src/Plugins/BotSharp.Plugin.RoutingSpeeder/Providers/IntentClassifier.cs

```diff
@@ -32,6 +32,10 @@ public class IntentClassifier
     public bool isModelReady => _isModelReady;
     private ClassifierSetting _settings;
 
+    private string[] _labels => GetLabels();
```

Comment:
`GetLabels()` will be always trigggered.

Reply:
Could I ask how to fix this?

Reply:
```csharp
private string[] _labels;
public string Labels => GetLabels();

private string[] GetLabels()
{
       if (_labels == null)
       {
           ... 
       }
       return _labels;
}
```